samples/Ark.ReferenceProject/Core/Ark.Reference.Core.Tests/Features/BookPrintProcess.feature

```diff
@@ -0,0 +1,48 @@
+Feature: BookPrintProcess
+    Background processing of book printing with Rebus messaging
+
+@CleanDbBeforeScenario
+Scenario: Create a book print process successfully
+    Given I have created a book with title "Test Book" and author "Test Author"
+    When I create a book print process for that book
+    Then the print process should be created with status "Pending"
+    And the print process progress should be 0
+
+@CleanDbBeforeScenario
+Scenario: Cannot create print process when one is already running
+    Given I have created a book with title "Test Book" and author "Test Author"
+    And I have created a book print process for that book
+    When I try to create another book print process for that book
+    Then the request fails with 400
+    And the business rule violation code is "BookPrintingProcessAlreadyRunningViolation"
+
+@CleanDbBeforeScenario
+Scenario: Print process completes successfully in background
+    Given I have created a book with title "Test Book" and author "Test Author"
+    And I have created a book print process for that book with ShouldFail false
+    When I wait background bus to idle and outbox to be empty
+    And I retrieve the print process status
+    Then the print process is
+    | Status    | Progress |
+    | Completed | 1.0      |
+
+@CleanDbBeforeScenario
+Scenario: Print process fails at 30% progress when ShouldFail is true
```

Comment:
remove this Scenario as the ShouldFail is a 'test helper' not a requirement.

the next Scenario is correct